samples/project-anfield/import_cluster_config.py

```diff
@@ -1281,7 +1411,9 @@ def create_protection_jobs():
                 exported_entity_mapping = (
                     cluster_dict["sql_entity_mapping"]
                     if environment == env_enum.KSQL
-                    else cluster_dict["ad_entity_mapping"] if environment == env_enum.KAD else cluster_dict["oracle_entity_mapping"]
+                    else cluster_dict["ad_entity_mapping"]
+                    if environment == env_enum.KAD
```

Comment:
I believe this will break if this is multiline ?

Reply:
Nope